src/optimization-engine/runbooks/data-collection/Export-ARGNICPropertiesToBlobStorage.ps1

```diff
@@ -79,8 +79,8 @@ switch ($authenticationOption) {
 if (-not($storageAccountSinkKey))
 {
     Write-Output "Getting Storage Account context with login"
-    Select-AzSubscription -SubscriptionId $storageAccountSinkSubscriptionId
-    $saCtx = (Get-AzStorageAccount -ResourceGroupName $storageAccountSinkRG -Name $storageAccountSink).Context
+    
+    $saCtx = New-AzStorageContext -StorageAccountName $storageAccountSink -UseConnectedAccount -Environment $cloudEnvironment
```

Comment:
nit: Seems like you have a bit of a pattern here. I'm not looking at it holistically, but it might be worth considering shared code to reduce duplication. No need to do this now, but something to consider in the future, especially as we look at AOEv2.

Reply:
Yes, absolutely. I thought of implementing a reusable approach, but hopefully this will be the last change to this piece of code, as we are focusing now on AOE v2.